pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/DoubleAggregationResultHolder.java

```diff
@@ -42,6 +42,11 @@ public void setValue(double value) {
     _value = value;
   }
 
+  @Override
+  public void setValue(int value) {
+    throw new RuntimeException("Method 'setValue' (with long value) not supported for class " + getClass().getName());
```

Comment:
(minor) same for other places
```suggestion
    throw new RuntimeException("Method 'setValue' (with int value) not supported for class " + getClass().getName());
```